packages/scss/src/components/_label.scss

```diff
@@ -30,3 +30,14 @@
 		}
 	}
 }
+
+.label {
+	&.mod-number {
+		border-radius: _theme("sizes.standard.line-height");
+		padding: _theme("spacings.reset");
+		height: _theme("sizes.standard.line-height");
```

Comment:
we might need to update others "label" to adopt this. But we might wait for the 4.2